### PR TITLE
Fix test failures (FoodCritic and Rubocop)

### PR DESCRIPTION
**Pull request by shortdudey123**

### CODE REVIEW

**Comment by devsibwarra** [comment]
👍

**Comment by shortdudey123** [comment]
rebased on master and fixed another rubocop offense that got in after i branched off

File: providers/configure.rb [hunk 1]
```diff
@@ -86,13 +86,13 @@ def configure
...
-    else
-      descriptors = current['maxclients']
-    end
+    descriptors = if current['ulimit'] == 0
+                    current['maxclients'] + 32
+                  elsif current['ulimit'] > current['maxclients']
+                    descriptors = current['ulimit']
+                  else
+                    descriptors = current['maxclients']
+                  end
```
Comment (devsibwarra) [inline providers/configure.rb]: I think you missed a few pieces here

Comment (shortdudey123) [inline providers/configure.rb]: fixed

**Comment by devsibwarra** [comment]
Looks good from my perspective

File: providers/configure.rb [hunk 2]
```diff
@@ -325,31 +325,21 @@ def configure
...
-           :requirepass => current['requirepass'],
-           :shutdown_save => current['shutdown_save'],
-           :save => current['save'],
-           :configdir => current['configdir'],
-           :piddir => piddir,
-           :platform => node['platform'],
-           :unixsocket => current['unixsocket']
-         })
-         only_if { node['redisio']['job_control'] == 'rcinit' }
-       end
```
Comment (brianbianco) [inline providers/configure.rb]: Am I missing something here, Why were so many variables removed from this?

Comment (devsibwarra) [inline providers/configure.rb]: I'm guessing they were unused in the template, which is a rubocop complaint I've had with my own code and a newer rubocop

Comment (shortdudey123) [inline providers/configure.rb]: The variables that were removed are unused and tripped [FC034](http://www.foodcritic.io/#FC034)

**Comment by brianbianco** [comment]
Ok thanks. I should pay better attention. I wonder why they were added in the first place. This looks good for a merge.

**Comment by shortdudey123** [comment]
No problem and cool

**Comment by shortdudey123** [comment]
@brianbianco any final thoughts before merging? :)

**Comment by martinb3** [comment]
LGTM 👍

**Comment by brianbianco** [comment]
Thanks again @shortdudey123

**Pull request merged**